derex/runner/templates/local.yml.j2

```diff
@@ -40,7 +41,8 @@ x-common-cms:
     &cms-env
     SERVICE_VARIANT: cms
     DEREX_PROJECT: {{ project.name }}
-    DJANGO_SETTINGS_MODULE: cms.envs.derex_project.{{ project.settings.name }}
+    # This will be used to compose the DJANGO_SETTINGS_MODULE
+    # environment variable in the docker entrypoint
```

Comment:
Upon reconsideration I'd like to put this back in. Or, even better, solve the problem outlined below in a better way than this.

The reason is that sometimes you might enter the container "from the side door", i.e. not through the entrypoint.
In particular just now I entered a container shell with `ddc-project exec lms sh`; when I tried to `./manage.py lms runserver 0:4701` I got an error, because the variable `DJANGO_SETTINGS_MODULE` was not set.

Maybe we could patch the `manage.py` script to derive the `DJANGO_SETTINGS_MODULE` variable from the `SETTINGS` var.